Day to day work/Project documentation.md

```diff
@@ -0,0 +1,43 @@
+## Project Documentation
+
+The documentation should be separated into a few levels of the project you are working on. That way, anyone can easily find the info they are looking for by searching specific files section.
+
+Those levels are shown in the following diagram:
+![](/img/project_documentation/documentation_chart.png)
+
+### Base documentation - readme
+The readme of the platform (i.e., Android, iOS, Flutter, etc.) should contain only chapters defined in the [mobile template readme](https://github.com/infinum/mobile-readme-template). The idea behind that readme is to have only crucial information regarding the project's platform.
+
+You can find the template here: [Mobile readme template](https://github.com/infinum/mobile-readme-template)
+
+### Platform wiki
+If you need to add anything extra that is not covered by the readme template, you should create a wiki (e.g., repo wiki on GitHub). 
+
+That wiki can contain topics such as:
+- Security storage usage
+- Project Specifics (e.g., Xcode, Android Studio)
+- Third-party integrations
+- Testing Setup
+- Pentest preparation
+- etc.
+
+This wiki should only contain platform-specific documentation.
```

Comment:
I would put the HPB [Android](https://github.com/infinum/android-hpb-mhpb/wiki) and [iOS](https://github.com/infinum/ios-hpb-mhpb/wiki) wiki here as examples. We should aim to have as many examples as we can for these kind of chapters.

Reply:
I agree with you! However, those two wikis are not available for everyone (only the dev on the project and TL/LE have access), so anyone trying to check them will see a 404 error :/ 

Should we create an example repo representing the Android and/or iOS HPB project? Maybe we can create something generic (copy content from HPB, replace HPB/mHPB/Asseco/... keywords with generic ones)?

Reply:
Added a generic explanation for getting examples of levels of documentation.